static/js/stream_edit.js

```diff
@@ -676,6 +676,15 @@ exports.initialize = function () {
         }
 
         subs.sub_or_unsub(sub);
+
+        if (
+            e.currentTarget.name === "subscription" &&
+            e.target.parentElement.parentElement.className === "empty_feed_notice"
```

Comment:
You do not need to check for target element name and also don't check using `class` of that banner, instead check using `id`.

Use `closest()` method in jQuery to check for the ancestor of target element (as used in other places) instead of using `parentElement` twice.